plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -369,7 +371,10 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
         int debugPort = getDebugPort();
         setupDebugTarget(devServerRunConfiguration, launch, debugPort, monitor);
       }
-      IVMInstall vmInstall = getVMInstall(configuration);
+      
+      IJavaProject javaProject = JavaCore.create(modules[0].getProject());
```

Comment:
You're right. What I mean though is that, if there is at least one module configured to use JDK 8, we should ensure we run the local server with JDK 8. (It won't be a problem to use JDK 8 when some of the modules use JDK 7, will it? Or, if it will be, we could show an error message that you can't mix Java7 and Java8 projects.)